### PR TITLE
Ping command and kostka command uprade

### PR DESCRIPTION
**Pull request by rkubapl**

### CODE REVIEW

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```
Comment (AnonymousX86) [inline .github/workflows/python-app.yml]: Project is using Python version 3.7.

Comment (rkubapl) [reply .github/workflows/python-app.yml]: ok

**Review by AnonymousX86 — CHANGES_REQUESTED** [review]
Change many things, mostly language and syntax.

**Review by rkubapl — COMMENTED** [review]
.

**Pull request merged**